src/dali/plugin/input/rest/kraken.py

```diff
@@ -230,8 +251,17 @@ def _compute_transaction_set(self, trade_history: Dict[str, Dict[str, str]], led
 
             if record[_TYPE] == _TRADE and not is_fiat_asset:
                 self.__logger.debug("Trade history record: %s", trade_history[record[_REFID]])
-                market: Dict[str, str] = self._client.markets_by_id[trade_history[record[_REFID]][_PAIR]]
-                asset_quote: str = market[_QUOTE]
+                markets: List[Dict[str, str]] = self._client.markets_by_id[trade_history[record[_REFID]][_PAIR]]  # type: ignore
+                if len(markets) > 1:
+                    possible_quotes = [market[_QUOTE] for market in markets]
+                    self.__logger.warning(
+                        "Multiple markets for pair. Which quote to use for %s market? Defaulting to quote=%s "
```

Comment:
Ditto: could you add a comment describing the reasons that can cause this warning?

Reply:
same reason as before. I can add a comment to address but should I keep this defensive code, change it to exception or just add a comment.

Reply:
Defensive code is great: same observation as above.